nedap/utils.test
# Deploy to clojars

## Brief

Let's make this library publicly available through clojars in preparation of `utils.spec` and `formatting-stack`.

I created a shared clojars user (`circleci-sts`) and added those credentials to the `JFrog`-context.

## QA plan

CircleCI should be able to build and deploy a tag. 
In order to do that someone (?) needs to add`circleci-sts` as a member of the `com.nedap.staffing-solutions`-group in clojars

## Author checklist

* [x] I have self-reviewed the PR prior to assignment, and its build passes
* [x] `circleci-sts` user has to be added as a member of clojars `com.nedap.staffing-solutions`-group on clojars.
* Specifically, I have code-reviewed iteratively the PR considering the following aspects in isolation:

## Reviewer checklist

* I have code-reviewed iteratively the PR considering the following aspects in isolation:
  * [ ] Business-facing correctness
  * [x] I added the `circleci-sts` user as a member of clojars `com.nedap.staffing-solutions`-group.
